cli/block/service.py

```diff
@@ -42,3 +42,33 @@ def get(self, identifier):
             return None, str(error)
 
         return block.get('data'), None
+
+    def get_list(self, ids, head, limit, reverse):
+        """
+        Get a list of blocks.
+
+        A list of blocks could be filtered by blocks identifiers, limit, reverse.
+
+        Arguments:
+            ids (list, optional): identifiers to get a list of blocks by.
+            head (string, optional): block identifier to get a list of transactions from.
```

Comment:
It also, transactions from - transactions to.

Reply:
@Alladin9393, done.